### PR TITLE
Clarify how to embed type info

### PR DESCRIPTION
**Pull request by andreasohlund**
@weralabaj did some tweaks, what do you think?

### CODE REVIEW

File: nservicebus/messaging/third-party-integration.md [hunk 1]
```diff
@@ -18,16 +18,17 @@ To get the best use of your tools ensure that the following information is inclu
...
-Refer to integration samples for [RabbitMQ](/samples/rabbitmq/native-integration/) and [SQL](/samples/sqltransport/native-integration/) transports, in order to see how full type name can be included in the payload.
+Embedding type info in the message body is currently supported by the default XmlSerializer and also the Json.NET seriliazer. The [RabbitMQ](/samples/rabbitmq/native-integration/) and [SQL](/samples/sqltransport/native-integration/) integration samples demonstrates this.
 
-Alternatively, include the following header in your message:
+If using a different serilizer or if you want to avoid embedding type info in your message bodies you can include the following header in your message:
 
 Header key  | Value
 ------------- | -------------
-NServiceBus.EnclosedMessageTypes  | Fully Qualified Message type including the assembly name, for e.g. IntegrationSample.Messages.Commands.ProcessOrder, IntegrationSample.Messages
+NServiceBus.EnclosedMessageTypes  | [FullName](https://msdn.microsoft.com/en-us/library/system.type.fullname(v=vs.110).aspx) of your message type, e.g. IntegrationSample.Messages.Commands.ProcessOrder
```
Comment (SimonCropp) [inline nservicebus/messaging/third-party-integration.md]: remove the version `(v=vs.110)`

Comment (SimonCropp) [inline nservicebus/messaging/third-party-integration.md]: make IntegrationSample.Messages.Commands.ProcessOrder code

Comment (andreasohlund) [inline nservicebus/messaging/third-party-integration.md]: done * 2

**Comment by weralabaj** [comment]
Looks good!

**Pull request merged**